notebooks/Shuvomoy - Benders decomposition.ipynb

```diff
@@ -390,7 +401,7 @@
     "if statusMasterProblem == :Optimal\n",
     "    fmCurrent = getValue(t)\n",
     "    xCurrent=Float64[]\n",
-    "    for i in 1:dimX\n",
+    "        for i in collect(1:dimX)\n",
```

Comment:
Because with out collect I am getting concatenation is deprecated use
collect() message.

On Mon, Jan 25, 2016 at 6:23 PM, Miles Lubin <notifications@github.com>
wrote:

> In notebooks/Shuvomoy - Benders decomposition.ipynb
> <https://github.com/JuliaOpt/juliaopt-notebooks/pull/17#discussion_r50773616>
> :
>
> > @@ -390,7 +401,7 @@
> >      "if statusMasterProblem == :Optimal\n",
> >      "    fmCurrent = getValue(t)\n",
> >      "    xCurrent=Float64[]\n",
> > -    "    for i in 1:dimX\n",
> > +    "        for i in collect(1:dimX)\n",
>
> Why is this changed to collect() here and in many other places?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaOpt/juliaopt-notebooks/pull/17/files#r50773616>.
>



-- 
Shuvomoy Das Gupta
Ph.D. Student
Systems Control Group
Department of Electrical & Computer Engineering
University of Toronto
http://scg.utoronto.ca/~shuvomoy.dasgupta/